ARM.CMSIS.pdsc

```diff
@@ -3224,6 +3224,8 @@ and 8-bit Java bytecodes in Jazelle state.
         <file category="library" condition="CM4_LE_GCC"      name="CMSIS/RTOS/RTX/LIB/GCC/libRTX_CM3_IFX.a" src="CMSIS/RTOS/RTX/SRC/GCC"/>
         <file category="library" condition="CM4_FP_LE_GCC"   name="CMSIS/RTOS/RTX/LIB/GCC/libRTX_CM4_IFX.a" src="CMSIS/RTOS/RTX/SRC/GCC"/>
         <!-- IAR -->
+        <file category="library" condition="CM4_LE_IAR"      name="CMSIS/RTOS/RTX/LIB/IAR/RTX_CM3_IFX.a" src="CMSIS/RTOS/RTX/SRC/IAR"/>
+        <file category="library" condition="CM4_FP_LE_IAR"   name="CMSIS/RTOS/RTX/LIB/IAR/RTX_CM4_IFX.a" src="CMSIS/RTOS/RTX/SRC/IAR"/>
```

Comment:
We never provided these libraries so far. RTX4 is deprecated. Why do you think its worth to add these?

Reply:
Its not about the library per se. Its about RTX4 at a whole. We do not maintain RTX4 any more. Instead we strongly recommend customers still using RTX4 to move to RTX5.